src/pair_hybrid_scaled.cpp

```diff
@@ -66,18 +73,35 @@ void PairHybridScaled::compute(int eflag, int vflag)
   // update scale values from variables where needed
 
   const int nvars = scalevars.size();
+  int atomscaleflag = 0;
   if (nvars > 0) {
     auto vals = new double[nvars];
+    auto vars = new int[nvars];
     for (int k = 0; k < nvars; ++k) {
       int m = input->variable->find(scalevars[k].c_str());
       if (m < 0)
         error->all(FLERR, "Variable '{}' not found when updating scale factors", scalevars[k]);
-      vals[k] = input->variable->compute_equal(m);
+
```

Comment:
Could pull this out of compute so it isn't repeated every timestep, but maybe it doesn't matter.

Reply:
This is necessary if the value of the variable changes between calls to `compute()`. I did not add this, it was there already.